src/metabase/core.clj

```diff
@@ -60,6 +59,12 @@
                                    setup-url
                                    "\n\n")))))
 
+(defn- -init-create-setup-token
```

Comment:
I know you didn't come up with the name `-init-create-setup-token` here but since you're touching it I noticed it. I think we should change the name to fit our coding conventions elsewhere. I think something like `create-setup-token-and-log-setup-url!` would be a 100x better name for this function.

1. We don't really prefix anything else with a hyphen no matter how secret/internal it is
2. We suffix DML type things with `!` (https://guide.clojure.style/#naming-unsafe-functions)
3. `-init-create-setup-token` doesn't really tell you much about what this function does